src/errors/test-run/templates.js

```diff
@@ -78,6 +78,12 @@ export default {
         ${escapeHtml(err.errMsg)}
     `),
 
+    [TYPE.uncaughtErrorInCustomDOMPropertyCode]: err => markup(err, `
+        An error occurred when trying to calculate a custom Selector property ${err.property}:
```

Comment:
`custom Selector DOM property`? I guess it's worth enclosing property name into quotes: `"${err.property}"`